typings/csstype/index.d.ts

```diff
@@ -8,5 +8,8 @@ declare module 'csstype' {
     '--unusable-height'?: string
     '--progress-time-left'?: string
     '--seek-time-left'?: string
+    '--cell-idx'?: number
   }
 }
+
+export {}
```

Comment:
Without this `export`, all of the csstype types were overriden with the values in this file, instead of being added to.

Reply:
That is strange! Can you add this comment in the code to note that? It's not at all the expected behaviour with interface merging (unless something has changed in TS recently, but it would be a big breaking change I think).